tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor.Tests/MockGitHubEventClient.cs

```diff
@@ -469,10 +472,14 @@ public override RulesConfiguration LoadRulesConfiguration(string? rulesConfigLoc
         // The mock won't be calling the actual service and since the method it's overriding is
         // async, the warning needs to be disabled.
 #pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-        public override async Task<List<string>> QueryAILabelService(IssueEventGitHubPayload issueEventPayload)
+        public override async Task<IssueTriageResponse> QueryAILabelService(IssueEventGitHubPayload issueEventPayload)
```

Comment:
```suggestion
        public override Task<IssueTriageResponse> QueryAILabelService(IssueEventGitHubPayload issueEventPayload)
```

You only need the `async` keyword if you use `await`.   This will let you remove the warning suppression.